updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc

```diff
@@ -6,20 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-[id="update-restricted-network-cluster-update-service"]
+To receive update recommendations for clusters in a disconnected environment, you can use the following procedures to install and configure a local copy of the OpenShift Update Service onto a cluster.
```

Comment:
Suggested text `To get similar user experience as connected clusters, you should use the following procedures to install and configure OpenShift update service in the disconnected environment`.